src/main/java/org/springframework/integration/dsl/kafka/Kafka.java

```diff
@@ -16,18 +16,19 @@
 
 package org.springframework.integration.dsl.kafka;
 
-import java.util.Properties;
+import java.util.regex.Pattern;
 
-import org.springframework.integration.dsl.support.Consumer;
-import org.springframework.integration.dsl.support.PropertiesBuilder;
-import org.springframework.integration.kafka.core.ConnectionFactory;
-import org.springframework.integration.kafka.core.Partition;
-import org.springframework.integration.kafka.listener.KafkaMessageListenerContainer;
-import org.springframework.integration.kafka.support.ZookeeperConnect;
-import org.springframework.util.Assert;
+import org.apache.kafka.common.TopicPartition;
+
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.core.ProducerFactory;
+import org.springframework.kafka.listener.AbstractMessageListenerContainer;
+import org.springframework.kafka.listener.config.ContainerProperties;
+import org.springframework.kafka.support.TopicPartitionInitialOffset;
 
 /**
- * Factory class for Kafka components.
+ * Factory class for Kafka-0.9, 0.10 components.
```

Comment:
I don't think we should have versions here; changes might not be needed for 0.11